### DISCUSSION TITLE
How to customise shortcuts in CP Editor?

### DISCUSSION THREAD

**Discussion by H4ckOm** [cpeditor/cpeditor#730] [Q&A] [answered]
Please tell can I configure own shortcuts like ctrl + c to copy a line and ctrl + x to cut the line?

**Comment by ouuan** [comment, marked as answer]
1. You don't need any configuration to use <kbd>Ctrl+C</kbd> to copy and use <kbd>Ctrl+X</kbd> to cut.
2. As for now, there are only a few customizable hotkeys. You can set them in [the Key Bindings section of the preferences](https://cpeditor.org/docs/preferences/key-bindings/).
